cron_jobs/caravan_cron.php

```diff
@@ -144,7 +157,7 @@ function hourlyCaravan(System $system, $debug = true): void
     } else {
         echo "Script running...<br>";
         foreach ($queries as $query) {
-            $system->db->query("LOCK TABLES `caravans` WRITE;");
+            $system->db->query("LOCK TABLES `caravans` WRITE, `region_locations` WRITE, `resource_logs` WRITE, `villages` WRITE;");
```

Comment:
Is it possible for other code to read a value from the table while this lock is active, then write it back to the table after this lock has expired?

Reply:
I think "LOCK" is a hard lock on read/write from these tables for the duration?

Reply:
Oh you're right I forgot `WRITE` here blocks all access, not just writes. Was thinking there was some kind of "READWRITE" lock 😅